configure.ac

```diff
@@ -2717,15 +2717,15 @@ then
 	then
 		with_libmysql="no ($with_mysql_config failed)"
 	else
-		AC_CHECK_LIB(mysqlclient, mysql_init,
+		AC_CHECK_LIB(c, mysql_init, # Hack to allow percona
```

Comment:
Looking at this today http://www.gnu.org/savannah-checkouts/gnu/autoconf/manual/autoconf-2.69/html_node/Libraries.html mentions `AC_SEARCH_LIBS` which might be a better option here.